scripts/vulkaninfo_generator.py

```diff
@@ -57,10 +58,10 @@
  */
 '''
 
-custom_formaters = '''
-std::ostream &operator<<(std::ostream &o, VkConformanceVersion &c) {
-    return o << std::to_string(c.major) << "." << std::to_string(c.minor) << "." << std::to_string(c.subminor) << "."
-             << std::to_string(c.patch);
+custom_formaters = r'''
+void DumpVkConformanceVersion(Printer &p, std::string name, VkConformanceVersion &c, int width = 0) {
```

Comment:
This is a required change to make vkconfig_output work due to the json nature of the output. Otherwise the conformance version would be an unquoted string and not be valid json.